spec/integration/plan_spec.rb

```diff
@@ -95,7 +95,9 @@
         it 'doesnt output without verbose flag' do
           result = run_cli(%W[plan run output::verbose --targets #{target}] + config_flags,
                            outputter: Bolt::Outputter::Human, project: @project)
-          expect(result).not_to match(/Hi, I'm Dave/)
+          #see yaml_plan_spec.rb
+          expect(config_flags).not_to include '--verbose' #:(
+          expect(output.string).not_to match(/Hi, I'm Dave/)
```

Comment:
Correcting previous verbose PR. I'm pretty sure we don't want `expect(result)` here. Not passing, but I think once we figure out this issue on the current PR then it'll fix both.